pandas/core/generic.py

```diff
@@ -6679,7 +6679,7 @@ def fillna(
                 return result if not inplace else None
 
             elif not is_list_like(value):
-                if not self._mgr.is_single_block and axis == 1:
```

Comment:
@jbrockmendel should this be fixed in the block manager fillna?

Reply:
> should this be fixed in the block manager fillna?

i dont think so, the BlockManager method doesn't have an axis keyword